app/addons/replication/controller.js

```diff
@@ -245,18 +194,18 @@ export default class ReplicationController extends React.Component {
   }
 
   onTabChange (section, url) {
-    Actions.changeTabSection(section, url);
+    this.props.changeTabSection(section, url);
```

Comment:
This line is throwing an error because the `changeTabSection` prop is not being set.